google-http-client/src/main/java/com/google/api/client/http/GenericUrl.java

```diff
@@ -126,14 +163,26 @@ public GenericUrl(URI uri) {
    * @since 1.14
    */
   public GenericUrl(URL url) {
+    this(url, false);
+  }
+
+  /**
+   * Constructs from a URL.
+   *
+   * @param url URL
+   * @param verbatim flag, to specify if URL should be used as is.
```

Comment:
ditto
also no period since it's not a complete sentence

Reply:
done